.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "ignorePatterns": [
+        "node_modules",
+        "generated",
+        "**/__tests__/*",
+        "**/__mocks__/*",
+        "Dangerfile.*",
+        "*.d.ts"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "project": "tsconfig.json",
+        "sourceType": "module"
+    },
+    "extends": [
+        "@pagopa/eslint-config/strong",
+    ],
+    "rules": {
+        "functional/prefer-readonly-type": "off",
+        "prefer-arrow/prefer-arrow-functions": "off",
+        "@typescript-eslint/no-use-before-define": "off",
+        "import/order": "off",
+        "@typescript-eslint/no-floating-promises": "off",
+        "prettier/prettier": "off",
+        "sort-keys": "off",
+        "@typescript-eslint/semi": "off",
+        "@typescript-eslint/explicit-member-accessibility": "off",
+        "jsdoc/newline-after-description": "off",
+        "arrow-body-style": "off"
```

Comment:
We can enable almost all this rules using auto fix eslint feature.
The following rules can remain disabled if require a deep refactory.
```
"prefer-arrow/prefer-arrow-functions": "off",
"sort-keys": "off",
"@typescript-eslint/no-use-before-define": "off",
```

Reply:
Or, we can disable them locally so that at least new implementations will cope with standard rules

Reply:
Example here: https://github.com/pagopa/io-functions-express/pull/16#pullrequestreview-628731997